src/main/jit/Compiler.cpp

```diff
@@ -48,6 +48,11 @@ using llvm::Value;
 
 namespace rho {
 namespace JIT {
+namespace {
+llvm::MDNode* CreateBranchWeights(llvm::LLVMContext& context) {
+  return llvm::MDBuilder(context).createBranchWeights(1000, 1);
```

Comment:
Make these values parameters, or better yet create functions BranchLikelyTaken() and BranchUnlikelyTaken()